### DISCUSSION TITLE
Why my contributions are not showing in repository graph

### DISCUSSION THREAD

**Discussion by ariannargesi** [vercel/next.js#63464] [Help]
### Summary

Hi 
recently I opened 3 pull request with around 10 commits in Next.js repository and they all merged with canary. But I can't see my my name in Next.js contribution grahp [here](https://github.com/vercel/next.js/graphs/contributors) even after changing the time range.

**Comment by icyJoseph** [comment]
When were these made? I think there are certain transactions in GitHub that take "some time" to update, like repository languages, etc, so it might be the case that the contribution graph for a project doesn't immediately update upon every single contribution.

Wait a bit, but if it has been several weeks already, then it might be something else.

**Comment by ariannargesi** [comment]
@icyJoseph It's been two weeks 
https://github.com/vercel/next.js/pull/62979
https://github.com/vercel/next.js/pull/62980

And this one is for last night
https://github.com/vercel/next.js/pull/63390

**Reply by icyJoseph** [reply]
🤔 interesting, if I go to https://github.com/vercel/next.js/graphs/contributors-data.json, then I see only 97 people, which I guess might correspond with some limitation of this API? The person in that list with least contributions has 12 contributions. You have 13 though, maybe just wait a bit more, there's probably some background job calculating this.

**Comment by ariannargesi** [comment]
It's been a lot and still I can't see my name
@icyJoseph

**Reply by ariannargesi** [reply]
Who should I talk to?
